src/store/reducers.js

```diff
@@ -50,7 +50,7 @@ function entityLoaded(state) {
                 name: 'patch',
                 component: PatchTab
             },
-            {
+            insights?.chrome?.isBeta() && {
```

Comment:
```suggestion
(!insights?.chrome?.isProd() || insights?.chrome?.isProd() && insights?.chrome?.isBeta()) && {
```


@karelhala - ROS service is not yet in GA but ROS tab is visible under  inventory detail page in public Prod. 
ROS should be visible everywhere except in public prod (i.e under GA)

Reply:
Sure thing!